libtsuba/include/tsuba/RDG.h

```diff
@@ -31,6 +31,8 @@ struct KATANA_EXPORT RDGLoadOptions {
   /// nullopt means the partition associated with the current host's ID will be
   /// loaded
   std::optional<uint32_t> partition_id_to_load;
+  /// Client is trying to load RDG into expected_num_hosts machines
+  std::optional<uint32_t> expected_num_hosts;
   /// List of node properties that should be loaded
```

Comment:
@scober, I assume this is not an abuse of your RDGLoadOptions class.

Reply:
Is there a reason to not use `getSystemNetworkInterface().Num`?

Reply:
There is no reason.  Can I blame the fact that I don't like global variables?  Maybe it was the phase of the moon.  Regardless, more discussion on this issue in #957, but I'm eliminating this change.